solanches/models.py

```diff
@@ -134,6 +141,17 @@ def get_destaques(cardapio_id):
         cardapio = Cardapio.get_by_id(cardapio_id)
         return cardapio.get("destaques")
 
+    @staticmethod
+    def remove_produto(cardapio_id, produto_id):
+        query = {"_id": cardapio_id}
+        cardapio = Cardapio.get_by_id(cardapio_id)
+        new_destaques = cardapio.get("destaques")
+        new_destaques.remove(produto_id) if produto_id in new_destaques else new_destaques
+        new_produtos = cardapio.get("produtos")
+        new_produtos.remove(produto_id)
```

Comment:
Essa verificação está sendo feita no controller, mas é importante que as camadas sejam independentes. Então, é bom verificar aqui também
```suggestion
        new_produtos.remove(produto_id) if produto_id in new_produtos else new_produtos
```